### PR TITLE
[LOCAL] Es additional translation strings

### PR DESCRIPTION
**Pull request by martinosmartinos**

### CODE REVIEW

File: languages/es_ES.po [hunk 1]
```diff
@@ -149,11 +149,11 @@ msgstr "bosques"
 
 #: Mongabay-News-Wordpress-Theme/components/lib/topics-section.php:59
 msgid "indigenous-peoples"
-msgstr "pueblos-indigenas"
+msgstr "pueblos-indígenas"
```
Comment (martinosmartinos) [inline languages/es_ES.po:152]: Lower case strings are actually url safe permalinks so it should remain 'pueblos-indigenas'. Correct topic name is fetched from data base using this slug.

File: languages/es_ES.po [hunk 2]
```diff
@@ -149,11 +149,11 @@ msgstr "bosques"
...
 
 #: Mongabay-News-Wordpress-Theme/components/lib/topics-section.php:59
 msgid "agroecology"
-msgstr "agroecologia"
+msgstr "agroecología"
```
Comment (martinosmartinos) [inline languages/es_ES.po:156]: Same idea here.

File: languages/es_ES.po [hunk 3]
```diff
@@ -165,7 +165,7 @@ msgstr "Océanos"
 #: Mongabay-News-Wordpress-Theme/components/lib/topics-section.php:59
 #: Mongabay-News-Wordpress-Theme/search.php:46 Mongabay-News-Wordpress-Theme/search.php:55
 msgid "oceans"
-msgstr "oceanos"
+msgstr "océanos"
```
Comment (martinosmartinos) [inline languages/es_ES.po:168]: Same as above, correct safe slug is oceanos.

File: languages/es_ES.po [hunk 4]
```diff
@@ -190,7 +190,7 @@ msgstr "Conservación"
 
 #: Mongabay-News-Wordpress-Theme/search.php:45 Mongabay-News-Wordpress-Theme/search.php:54
 msgid "conservation"
-msgstr "conservacion"
+msgstr "conservación"
```
Comment (martinosmartinos) [inline languages/es_ES.po:193]: Same as above. Should be unchanged.

**Comment by WillieShubert** [comment]
Hey @martinosmartinos , do I need to take action? I understand your logic for removing the accent marks.

**Pull request merged**